scripts/build_arrow_source.sh

```diff
@@ -26,7 +26,7 @@ ARROW_DEPS_BUILD_DIR=$DEPENDENCY_DIR/arrow_deps
 ARROW_CMAKE_BUILD_DIR=$ARROW_SOURCE_DIR/cpp/cmake-build
 
 GIT_REPO="https://github.com/apache/arrow.git"
-CLONE_CMD="git clone -b master $GIT_REPO $ARROW_SOURCE_DIR"
+CLONE_CMD="git clone $GIT_REPO $ARROW_SOURCE_DIR"
```

Comment:
The arrow project (https://github.com/apache/arrow) renamed their `master` branch to `main`.  I suspect the default branch is fine for the clone target given subsequent steps will checkout a versioned tag.